manifests/backup/files.pp

```diff
@@ -0,0 +1,70 @@
+# @summary Run a backup for files
+#
+# @param paths
+#   The file paths to backup
+# @param excludes
+#   The file paths to exclude form the backup
+# @param password
+#   The password on the repository
+# @param repository
+#   The repository to backup to
+#
+# @example Back up /home and /etc to target
+#   restic::backup::files { 'target':
+#     paths      => ['/home', '/etc'],
+#     repository => 'sftp:USERNAME@HOST:restic-repo',
+#   }
+#
+define restic::backup::files (
+  Array[Stdlib::Absolutepath, 1] $paths,
+  Variant[String[1], Sensitive[String[1]]] $password,
+  Variant[String[1], Sensitive[String[1]]] $repository,
+  Array[Stdlib::Absolutepath] $excludes = [],
+) {
+  require restic::files
+
+  $sh_file = "/etc/restic/${name}.sh"
```

Comment:
what is this file used for? I don't see any reference to it besides the generation.

Reply:
I think it was a left over of when I sourced the variables in a shell script, before using systemd env vars. Will drop.